application.properties

```diff
@@ -1,5 +1,5 @@
 #Grails Metadata file
-app.grails.version=2.5.1
+app.grails.version=2.5.0
```

Comment:
Grails 2.5.1 contains stability and security fixes we will be using this going forward.
Also both build systems are set to use 2.5.1, this line will cause the build to fail.